### PR TITLE
Fix for _.isString and _.isPlainObject

### PR DESCRIPTION
**Pull request by Wallacy**
This fixes: #740, and fix: #741

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -73,10 +72,18 @@ const utils = {
...
+		return typeof s === "string";
+	},
+
 	isObject(o) {
 		return o !== null && typeof o === "object";
 	},
 
+	isPlainObject(o) {
+		return o !=null ? Object.getPrototypeOf(o) === Object.prototype || Object.getPrototypeOf(o) === null : false;
+	},
```
Comment (icebob) [inline src/utils.js:85]: Please add unit tests for them.

Comment (Wallacy) [reply src/utils.js:85]: Ops... I forget that.

Comment (Wallacy) [reply src/utils.js:85]: Done... 

Doing the tests a observed the isObject test was incomplete, then i added more tests. Doing that i observed that object-wrapped strings was not handled and i added a fix to that;

Comment (icebob) [reply src/utils.js:85]: Great, thanks!

**Pull request merged**